src/main/java/ai/pluggy/client/response/Transaction.java

```diff
@@ -15,4 +15,6 @@
   String providerCode;
   TransactionStatus status;
   PaymentData paymentData;
+  // "CREDIT", "DEBIT"
+  String type;
```

Comment:
Since we know and can enumerate the possible values in advance, let's use an enum type instead, so the value itself becomes self-documenting as well.